### ISSUE TITLE
Patched Symbol font are always centered

### ISSUE THREAD

**Issue by mangelozzi** [ryanoasis/nerd-fonts#513]
# Intro
In Unicode there are two symbols often used for drawings boxes:
Left Half Block: <https://www.compart.com/en/unicode/U+258C>
Right Half Block: <https://www.compart.com/en/unicode/U+2590>

1. Left Half Block:  ▌▌▌
2.  Right Half Block: ▐▐▐

Its really hard to get a Nerd patched font with these symbols, even though they are common block drawing elements.

When these symbols are patched from another font with `--custom`, it centers them, making them useless.

**Comment by Frederick888** [comment]
I can patch Bitstream using a [script](https://github.com/Frederick888/bitstream-vera-nerd-hybrid-mono) I wrote.

![image](https://user-images.githubusercontent.com/4507647/90319416-34eadc80-df7b-11ea-80f4-d21dd71a86fd.png)

**Comment by mangelozzi** [comment]
Thanks Frederick. I maanged to use the original script by changing it as follows:
```
        CUSTOM_ATTR = {
            # 'pa' == preserve aspect ratio
            'default': {'align': '', 'valign': '', 'stretch': '', 'params': ''},

            # Don't center these block off center block elements vertically.
            # e.g. Upper half block
            0x2580: {'align': 'c', 'valign': '', 'stretch': 'pa', 'params': ''},
            0x2581: {'align': 'c', 'valign': '', 'stretch': 'pa', 'params': ''},
            0x2582: {'align': 'c', 'valign': '', 'stretch': 'pa', 'params': ''},
            0x2583: {'align': 'c', 'valign': '', 'stretch': 'pa', 'params': ''},
            0x2584: {'align': 'c', 'valign': '', 'stretch': 'pa', 'params': ''},
            0x2585: {'align': 'c', 'valign': '', 'stretch': 'pa', 'params': ''},
            0x2586: {'align': 'c', 'valign': '', 'stretch': 'pa', 'params': ''},
            0x2587: {'align': 'c', 'valign': '', 'stretch': 'pa', 'params': ''},
            0x2594: {'align': 'c', 'valign': '', 'stretch': 'pa', 'params': ''},

            # Don't center these block off center block elements horizontally.
            # e.g. Left half block
            0x2589: {'align': '', 'valign': 'c', 'stretch': 'pa', 'params': ''},
            0x258a: {'align': '', 'valign': 'c', 'stretch': 'pa', 'params': ''},
            0x258b: {'align': '', 'valign': 'c', 'stretch': 'pa', 'params': ''},
            0x258c: {'align': '', 'valign': 'c', 'stretch': 'pa', 'params': ''},
            0x258d: {'align': '', 'valign': 'c', 'stretch': 'pa', 'params': ''},
            0x258e: {'align': '', 'valign': 'c', 'stretch': 'pa', 'params': ''},
            0x258f: {'align': '', 'valign': 'c', 'stretch': 'pa', 'params': ''},
            0x2590: {'align': '', 'valign': 'c', 'stretch': 'pa', 'params': ''},
            0x2595: {'align': '', 'valign': 'c', 'stretch': 'pa', 'params': ''},

            # Don't center these block off center block elements horizontally or vertically.
            # e.g. Quadrant lower left
            # These have mixed alignment rules
            0x2596: {'align': '', 'valign': '', 'stretch': 'pa', 'params': ''},
            0x2597: {'align': '', 'valign': '', 'stretch': 'pa', 'params': ''},
            0x2598: {'align': '', 'valign': '', 'stretch': 'pa', 'params': ''},
            0x259d: {'align': '', 'valign': '', 'stretch': 'pa', 'params': ''},
        }
```

**Comment by Finii** [comment]
This will be solved (become unneeded) with #1125
